### PR TITLE
Week 2 Step 2 Done

### PR DESCRIPTION
**Pull request by SofiaPSU**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -20,9 +20,9 @@ <h1>Sofia Riley</h1>
       <p> On campus I am involved with the Assocation of Women in Computing, the Penn State Equestrian Team, the Girls Who Code College Loop, and the Women in Engineering Program.</p>
       <p> I am a Technology Lead for the Girls Who Code College Loop and a Facilitator for the study groups organized by the Women in Engineering Program.</p>
       
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Click here to get a phrase:</p>
+      <button onclick="showPhrase()">Phrase</button>
+      <div id="phrase-container"></div>
       <img style ='position:absolute; top:25px; left: 25px; width:300px; height:250px' src= "images/profile.jpg">
```
Comment (claudiapao10) [inline portfolio/src/main/webapp/index.html:26]: I like how everything is looking. I don't have a lot of knowledge in HTML or CSS, but I think you can put the img style on the stylesheet. Other than that, everything is great.

**Review by claudiapao10 — APPROVED** [review]
Everything is looking good.

File: portfolio/src/main/java/com/google/sps/servlets/NewServlet.java [hunk 2]
```diff
@@ -8,12 +8,12 @@
 import javax.servlet.http.HttpServletResponse;
 
 
-@WebServlet("/servlet")
+@WebServlet("/step_one_servlet")
```
Comment (GoogleyGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/NewServlet.java:11]: Try a more descriptive name -- step_one_servlet does not let me know the function of this endpoint.

File: portfolio/src/main/java/com/google/sps/servlets/NewServlet.java [hunk 3]
```diff
@@ -8,12 +8,12 @@
...
 public class NewServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Is it working?</h1>");
+    response.getWriter().println("Is it working?");
```
Comment (GoogleyGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/NewServlet.java:17]: Try to not leave debug messages in code that is being merged (unless this is what you wanted it to say).

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (GoogleyGabriel) [reply portfolio/src/main/webapp/index.html:26]: +1 to what Claudia said

**Review by GoogleyGabriel — CHANGES_REQUESTED** [review]
Looking great :)

**Pull request closed without merging**